Flaky TestSystemConnector.testQueryDuringAnalysisIsCaptured

```
2022-08-12T16:00:50.9087936Z [ERROR]   TestSystemConnector.testQueryDuringAnalysisIsCaptured:232->AbstractTestQueryFramework.assertQueryEventually:273 For query: 
2022-08-12T16:00:50.9088752Z  SELECT state FROM system.runtime.queries WHERE query LIKE '%test_query_id_2%' AND query NOT LIKE '%system.runtime.queries%'
2022-08-12T16:00:50.9089108Z not equal
2022-08-12T16:00:50.9089402Z Actual rows (up to 100 of 1 extra rows shown, 1 rows in total):
2022-08-12T16:00:50.9089664Z     [FAILED]
2022-08-12T16:00:50.9089952Z Expected rows (up to 100 of 1 missing rows shown, 1 rows in total):
2022-08-12T16:00:50.9090220Z     [FINISHED]
```

--- a/testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java
+++ b/testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java
@@ -432,6 +432,7 @@ public class DistributedQueryRunner
     {
         long start = System.nanoTime();
         coordinator.createCatalog(catalogName, connectorName, properties);
+        backupCoordinator.ifPresent(backup -> backup.createCatalog(catalogName, connectorName, properties));
         log.info("Created catalog %s in %s", catalogName, nanosSince(start));
     }
 

